src/pages/DetailsPage.spec.js

```diff
@@ -5,8 +5,8 @@ describe('DetailsPage', () => {
   it('renders number of headings and an images of the page', () => {
     render(<DetailsPage />)
 
-    // const title = screen.getAllByRole('heading')
-    // expect(title).toHaveLength(4)
+    const title = screen.getAllByRole('heading')
+    expect(title).toHaveLength(4)
```

Comment:
Im not sure, if you have implemented this test yet. But if so this test could need a bit more.